source/Classroom/Troubleshooting/Account_Administration/dont_know_your_username.md

```diff
@@ -15,8 +15,8 @@ Your SendGrid username can be any string of characters that works for you! If yo
 
 If you are unsure what your username is, consider checking your existing code or SMTP integration, as your username will probably be in there plain as day.
 
-If you still can't find your username, get in touch with [support](https://support.sendgrid.com/hc/en-us), and we can help you track it down!
+If you still can't find your username, [complete this form](https://support.sendgrid.com/hc/en-us/requests/new#forgot-username) to begin the recovery process, and we can help you track it down!
```

Comment:
If you still can’t find your username, [complete this form](https://support.sendgrid.com/hc/en-us/requests/new#forgot-username to begin the username recovery process.